hack/windows/Prepare-Node.ps1

```diff
@@ -45,9 +35,16 @@ function DownloadFile($destination, $source) {
     }
 }
 
-If (Get-Service kubelet -ErrorAction SilentlyContinue) {
-    Write-Host("Found existing kubelet service, exit.")
-    exit 0
+if ($ContainerRuntime -eq "docker") {
+    if (-not(Test-Path "//./pipe/docker_engine")) {
+        Write-Error "Docker service was not detected - please install and start Docker before calling Prepare-Node.ps1 with -ContainerRuntime docker"
+        exit 1
+    }
+} elseif ($ContainerRuntime -eq "containerd") {
```

Comment:
In your example in windows.md, you are using "Containerd", but here we check with "containerd", is the capital char ignored?

Reply:
Made this change